trade-finance-manager-api/src/generateApp.js

```diff
@@ -29,7 +30,7 @@ const generateApp = () => {
   app.use(removeCsrfToken);
   app.use(healthcheck);
   app.use(passport.initialize());
-  app.post('/v1/login', loginController.login);
+  app.post('/v1/login', validateSsoFeatureFlagIsOff, loginController.login);
```

Comment:
Can we use `isTfmSsoFeatureFlagEnabled` instead? Other feature flag checks have this convention:

```
isTfmDealCancellationFeatureFlagEnabled
isTfmFacilityEndDateFeatureFlagEnabled
isTfmPaymentReconciliationFeatureFlagEnabled
```

Reply:
Resolving this as I can see why we want to do this now => https://github.com/UK-Export-Finance/dtfs2/pull/3687#discussion_r1814745990